network/core-node/admin-guide/prerequisites.mdx

```diff
@@ -3,32 +3,79 @@ title: Prerequisites
 
-- **Inbound**: a Stellar Core node needs to allow all IPs to connect to its `PEER_PORT` over TCP. You can specify a port when you [configure Stellar Core](./configuring.mdx), but most people use the default, which is **11625**.
-- **Outbound**: a Stellar Core needs to connect to other nodes via their `PEER_PORT`s TCP. You can find information about other nodes' `PEER_PORT`s on a network explorer like [Stellarbeat](https://stellarbeat.io/), but most use the default port, which is, again, **11625**.
+### Inbound
+
+A Stellar Core node needs to allow all IPs to connect to its `PEER_PORT` over TCP. You can specify a port when you [configure] Stellar Core, but most people use the default, which is **11625**.
+
+### Outbound
+
+A Stellar Core node needs to connect to other nodes on the internet via their `PEER_PORT` over TCP. You can find information about other nodes' `PEER_PORT`s on a network explorer like [Stellarbeat](https://stellarbeat.io/), but most use the default port for this as well, which is (again) **11625**.
 
 ## Internal System Access
 
-Stellar Core also needs to connect to certain internal systems, though exactly how varies based on your setup.
+Stellar Core also needs to connect to certain internal systems, though exactly how this is accomplished can vary based on your setup.
+
+### Inbound
+
+- Stellar Core exposes an _unauthenticated_ HTTP endpoint on its `HTTP_PORT`. You can specify a port when you [configure] Stellar Core, but most people use the default, which is **11626**.
+- The `HTTP_PORT` is used by other systems (such as Horizon) to submit transactions, so this port may have to be exposed to the rest of your internal IP addresses.
+- It's also used to query Stellar Core [info](./commands.mdx) and provide [metrics](./monitoring.mdx).
+- And to perform administrative commands such as [scheduling upgrades](./network-upgrades.mdx) and changing log levels
+- For more on that, see [commands](./commands.mdx)
+
+:::note Note on exposing the HTTP endpoint
+
+If you need to expose this endpoint to other hosts in your local network, we strongly recommended you use an intermediate reverse proxy server to implement authentication. Don't expose the HTTP endpoint to the raw and cruel open internet.
+
+:::
+
+### Outbound
+
+- Stellar Core requires access to a database (PostgreSQL for example). If that database resides on a different machine on your network, you'll need to allow that connection. You'll specify the database when you [configure] Stellar Core.
+- You can safely block all other connections.
+
+## Storage
+
+Stellar Core stores two copies of the ledger: one in a SQL [database](#database) and one in XDR files on local disk called [buckets](#buckets). For the most part, the contents of both the database and buckets directories can be ignored, since they are completely managed by Stellar Core. In terms of storage space, 100 GB is enough in 2024. Most people seem to have at least 1TB on hand.
```

Comment:
What does "most people have at least 1 TB on hand" mean?  It feels a bit hand wavy.  Are we confident that's accurate?  If so, can we provide a reason?  This isn't necessarily true, but here's an example of what I mean: "Most people seem to have at least 1TB on hand because they like to have a ton of headroom just in case, and there's not a big cost difference between 1 TB and 100 GB."

Reply:
I think this is a phrase that was already in this docs repo (and I can't find a corollary in the [gh admin guide](https://github.com/stellar/stellar-core/blob/master/docs/software/admin.md). As I was combining things, my mind was thinking pretty similarly to the example reasoning you provided. Never really questioned my own assumption.

It seems to me like a useless detail that doesn't really add anything meaningful to the conversation. I think we ought to take it out.